encord/objects/ontology_labels_impl.py

```diff
@@ -245,6 +245,19 @@ def priority(self) -> Optional[float]:
 
         return self._label_row_read_only_data.priority
 
+    @property
+    def label_is_valid(self) -> bool:
```

Comment:
Definitely matter of taste, but since this is a property of the LabelRow object, having "label_" prefix in the name looks redundant. Should we consider renaming to just "is_valid" instead?